activities/activity_johnpolo.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+"""
+Instructions
+- Functions intended to run for each scan
+  need to be named run_xxxxx
+
+- Do not modify the parameters of the run_xxx function
+  unless you know what you are doing
+  see optional parameters:
+  https://github.com/tangible-landscape/grass-tangible-landscape/wiki/Running-analyses-and-developing-workflows#python-workflows
+
+- All gs.run_command/read_command/write_command/parse_command
+  need to be passed env parameter (..., env=env)
+"""
+
+import grass.script as gs
+
+
+def run_topo_par(scanned_elev, morphmap, select_meth, env, **kwargs):
+    gs.run_command(
+        "r.param.scale",
+        input=scanned_elev,
+        output=morphmap,
+        method=select_meth,
+        env=env,
+        overwrite=True,
+    )
+
+
+# this part is for testing without TL
+def main():
+    import os
+
+    # get the current environment variables as a copy
+    env = os.environ.copy()
+    # we want to run this repetetively without deleted the created files
+    env["GRASS_OVERWRITE"] = "1"
+
+    elevation = "elev_lid792_1m"
+    select_meth = "profc"
+
+    run_topo_par(
+        scanned_elev=elevation,
+        morphmap="feat_verticalcurv",
+        env=env,
+        select_meth=select_meth,
```

Comment:
`select_meth` and `morphmap` need to be hardcoded in the `run_topo_par` function. In general, this would be a bad design, but here it is actually required by the Tangible Landscape API (which wouldn't know what to assign to these parameters).

Reply:
I edited the `gs.run.command` by adding: `output="feat_verticalcurv",` and `method="profc",`. This is how I understood what you mean by hardcoding the function. Hopefully, I got it right this time. :)

Reply:
Yes. That's it!